html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <label for="email">Email:</label>
+
+      <input type="email" id="email" size="30" required>
+      <form action="http://www.example.com/profile.php">
```

Comment:
you want to make sure that there is only one `form` element in the document, and it must surround all inputs and labelss